docs/UsersGuide/source/BuildRunSRW.rst

```diff
@@ -0,0 +1,883 @@
+   | Maximum attempt      | MAXTRIES_MAKE_GRID, MAXTRIES_MAKE_OROG,                    |
+   |                      | MAXTRIES_MAKE_SFC_CLIMO, MAXTRIES_GET_EXTRN_ICS,           |
+   |                      | MAXTRIES_GET_EXTRN_LBCS, MAXTRIES_MAKE_ICS,                |
+   |                      | MAXTRIES_MAKE_LBCS, MAXTRIES_RUN_FCST, MAXTRIES_RUN_POST   |
+   +----------------------+------------------------------------------------------------+
+   | Post configuration   | USE_CUSTOM_POST_CONFIG_FILE, CUSTOM_POST_CONFIG_FP         |
+   +----------------------+------------------------------------------------------------+
+   | Running ensembles    | DO_ENSEMBLE, NUM_ENS_MEMBERS                               |
+   +----------------------+------------------------------------------------------------+
+   | Stochastic physics   | DO_SHUM, DO_SPPT, DO_SKEB, SHUM_MAG, SHUM_LSCALE,          |
+   |                      | SHUM_TSCALE, SHUM_INT, SPPT_MAG, SPPT_LSCALE, SPPT_TSCALE, |
+   |                      | SPPT_INT, SKEB_MAG, SKEB_LSCALE, SKEP_TSCALE, SKEB_INT,    |
+   |                      | SKEB_VDOF, USE_ZMTNBLCK                                    |
+   +----------------------+------------------------------------------------------------+
+   | Boundary blending    | HALO_BLEND                                                 |
+   +----------------------+------------------------------------------------------------+
+   | FVCOM                | USE_FVCOM, FVCOM_DIR, FVCOM_FILE                           |
+   +----------------------+------------------------------------------------------------+
+   | Compiler             | COMPILER                                                   |
+   +----------------------+------------------------------------------------------------+
+
+
+.. _UserSpecificConfig:
+
+User-specific configuration: ``config.sh``
+--------------------------------------------
+
+The user must specify certain basic information about the experiment in a ``config.sh`` file located in the ``ufs-srweather-app/regional_workflow/ush`` directory. Two example templates are provided in that directory: ``config.community.sh`` and ``config.nco.sh``. The first file is a minimal example for creating and running an experiment in the *community* mode (with ``RUN_ENVIR`` set to ``community``). The second is an example for creating and running an experiment in the *NCO* (operational) mode (with ``RUN_ENVIR`` set to ``nco``).  The *community* mode is recommended in most cases and will be fully supported for this release. The operational/NCO mode will typically be used by those at the NOAA/NCEP/Environmental Modeling Center (EMC) and the NOAA/Global Systems Laboratory (GSL) working on pre-implementation testing. :numref:`Table %s <ConfigCommunity>` shows the configuration variables, along with their default values in ``config_default.sh`` and the values defined in ``config.community.sh``.
+
```

Comment:
".... implementation testing [for the Rapid-Refresh Forecast System (RRFS)].  You could add the bracketed part if you wanted.